README.md

```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
 
 The following subprojects are owned by the Working Group:
 
-{{
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
+    * https://github.com/RobotWebTools/rwt_dockers
+    * https://github.com/RobotWebTools/robot_web_tools
```

Comment:
This was last updated 2014 - it doesn't seem used. I'm not sure if it provides value? Should we remove it

Reply:
It was a sort of a meta package that includes all launchers for backend system. tf2_web_republihser, web_video_server and rosbridge. 

But, I agree to remove it. Let's keep the org. simple.